packages/theme/src/boxShadows.ts

```diff
@@ -0,0 +1,57 @@
+export const boxShadows = {
+    small: '0px 2px 4px rgba(0, 0, 0, 0.04)',
+    // TODO: next shadows needs to be defined
+    medium: '0px 2px 4px rgba(0, 0, 0, 0.04)',
+    big: '0px 2px 4px rgba(0, 0, 0, 0.04)',
+};
+
+export type BoxShadow = keyof typeof boxShadows;
+export type BoxShadows = typeof boxShadows;
+
+interface NativeBoxShadowDefinition {
+    elevation: number;
+    shadowColor: string;
+    shadowOffset: {
+        height: number;
+        width: number;
+    };
+    shadowOpacity: number;
+    shadowRadius: number;
+}
+
+export const nativeBoxShadows = {
```

Comment:
can this be typed too? `Record<string, NativeBoxShadowDefinition>`

Reply:
yea, why not